frontend/packages/console-shared/src/components/quick-search/QuickSearchList.scss

```diff
@@ -20,10 +20,10 @@
       }
     }
     &--highlight {
-      background: var(--pf-global--active-color--200) !important;
+      background: var(--pf-global--BackgroundColor--400) !important;
```

Comment:
We need to use `--pf-global--BackgroundColor--200` here. As highlight is not visible for light theme.
```suggestion
      background-color: var(--pf-global--BackgroundColor--200) !important;
```
![Peek 2022-03-30 16-34](https://user-images.githubusercontent.com/2561818/160821201-aa9580e7-b019-48aa-b38c-f10447606a6a.gif)

Reply:
fixed